README.md

```diff
@@ -25,90 +20,55 @@ Here are the method signatures as definied in the `StringPreference` class.
 
 ```java
 public class StringPreference {
-    public String get() { ... }
-    public boolean isSet() { ... }
-    public void set(String value) { ... }
-    public void delete() { ... }
+    public String get() {  }
+    public boolean isSet() {  }
+    public void set(String value) {  }
+    public void delete() {  }
 }
 ```
 
-Wrappers for frequently used classes can be found and added in the [wiki][typedpreferences-wiki]. Take a look!
+## Source
 
+Inspired by https://github.com/johnjohndoe/TypedPreferences - version 1.0
```

Comment:
Followthe ReadMe template shared.

Reply:
Why referring to v1.0, why not the latest v2.1.0 version of this library ? Check this